paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/EditPaymentViewState.kt

```diff
@@ -0,0 +1,17 @@
+package com.stripe.android.paymentsheet.ui
+
+import com.stripe.android.core.strings.ResolvableString
+import com.stripe.android.uicore.elements.SingleChoiceDropdownItem
+
+internal data class EditPaymentViewState(
+    val last4: String,
+    val canUpdate: Boolean,
+    val selectedBrand: CardBrandChoice,
+    val availableBrands: List<CardBrandChoice>
+) {
+    data class CardBrandChoice(
+        val id: String,
+        override val label: ResolvableString,
+        override val icon: Int
```

Comment:
Can we simplify this and keep a reference to `CardBrand`, with `label` and `icon` being computed properties? Would also make the code in the view model simpler.